src/web/MarkdownTextInput.css

```
@@ -0,0 +1,25 @@
+.react-native-live-markdown-input-single-line[contenteditable='true'] {
```

Looks like [here](https://www.w3schools.com/css/css_attribute_selectors.asp) they use `"` instead of `'` so let's change it everywhere in this file:
```suggestion
.react-native-live-markdown-input-single-line[contenteditable="true"] {
```

---

Prettier changes it to single quote `'` after saving